packages/babel-plugin-transform-jsx-to-htm/index.mjs

```diff
@@ -86,50 +86,24 @@ export default function jsxToHtmBabelPlugin({ types: t }, options = {}) {
 			cooked: buffer
 		}));
 		buffer = '';
-	}
-
-	function processNode(node, path, isRoot) {
-		const open = node.openingElement;
-		const { name } = open.name;
-
-		if (name.match(/^[A-Z]/)) {
-			raw('<');
-			expr(t.identifier(name));
-		}
-		else {
-			raw('<');
-			raw(name);
-		}
-
-		if (open.attributes) {
-			for (let i = 0; i < open.attributes.length; i++) {
-				const attr = open.attributes[i];
-				raw(' ');
-				if (t.isJSXSpreadAttribute(attr)) {
-					raw('...');
-					expr(attr.argument);
-					continue;
-				}
-				const { name, value } = attr;
-				raw(name.name);
-				if (value) {
-					raw('=');
-					if (value.expression) {
-						expr(value.expression);
-					}
-					else if (t.isStringLiteral(value)) {
-						escapePropValue(value);
-					}
-					else {
-						expr(value);
-					}
-				}
-			}
-		}
-
-		const children = t.react.buildChildren(node);
+  }
+  
+  function getName(node) {
+    switch (node.type) {
+      case 'JSXMemberExpression':
+        return `${node.object.name}.${node.property.name}`
+    
+      default:
+        return node.name;
+    }
+  }
+
+  function processChildren(node, name, isFragment) {
+    const children = t.react.buildChildren(node);
 		if (children && children.length !== 0) {
-			raw('>');
+      if (!isFragment) {
+        raw('>');
+      }			
```

Comment:
The original code uses tabs for indentation, so would it be possible to change the modified parts to do so also?

Reply:
Done